test/functional/launch/shared_mem/CMakeLists.txt

```diff
@@ -18,13 +18,15 @@ set(SHARED_MEM_TYPES DynamicMem StaticMem)
 
 foreach( BACKEND ${LAUNCH_BACKENDS} )
   foreach( SHARED_MEM_TYPES ${SHARED_MEM_TYPES} )
+    if( NOT ((BACKEND STREQUAL "Sycl") OR (SHARED_MEM_TYPES STREQUAL "StaticMem")))
```

Comment:
Is the intention to disable just the sycl staticmem test?

Reply:
I think this disables all sycl and staticMem tests.

Reply:
good catch, just pushed the fix. It should be AND